src/vuex/modules/editor.js

```diff
@@ -53,7 +53,9 @@ const mutations = {
     const tab = {
       ...payload,
       ...renderHTML(payload),
-      wordCount: wordCount(payload.content)
+      wordCount: wordCount(payload.content),
+      charCount: payload.content.length,
+      charCountWithoutSpace: payload.content.replace(/\s/g, '').length
```

Comment:
oh, i think we don't need to calculate `wordCount` in store, maybe move them to `footer.vue` as getters?